typespec/main.tsp

```diff
@@ -128,4 +129,10 @@ namespace Api {
 
     @route("/files")
     interface FilesRoute extends Files.Endpoints {}
+
+    @route("/developers/{id}/secrets")
```

Comment:
we should remove the `/developers/{id}` prefix because `developer_id` is already implicitly set inside the auth key.